src/index.js

```diff
@@ -30,7 +38,7 @@ const ACTION_UA = `${pkg.name}/${pkg.version}`;
     }
 
     // Set the PR # (if available) and branch name
-    !!pullRequestNumber && core.exportVariable('PERCY_PULL_REQUEST', pullRequestNumber);
+    !!pullRequestNumber && setPercyBranchBuildInfo(pullRequestNumber);
```

Comment:
Rather than a helper function that takes a PR number from the payload, then references the payload again for the ref, could we just conditionally set the two vars inline here like the other vars above? 

```suggestion
if (github.context.payload.number) {
  core.exportVariable('PERCY_PULL_REQUEST', github.context.payload.number);
  core.exportVariable('PERCY_BRANCH', github.context.payload.pull_request.head.ref);
}
```

If you think a helper is better, could we make it more centered around the payload rather than parts of the payload?

```js
// this would perform the same condition as the suggested change
setPercyBranchBuildInfo(github.context.payload)
```